extensions/wikia/Recirculation/services/ParselyDataService.class.php

```diff
@@ -52,16 +53,16 @@ function() use ( $type, $metadata, $count ) {
 	 */
 	private function apiRequest( $type, $meta, $count ) {
 		$options = [];
-		$options['limit'] = $count + self::PARSELY_API_EXTRA;
+		$options['limit'] = $count + static::PARSELY_API_EXTRA;
 
 		switch ( $type ) {
 			case 'vertical':
-				$options['sort'] = self::PARSELY_API_SORT;
+				$options['sort'] = static::PARSELY_API_SORT;
 				$options['pub_days'] = 30;
 				$endpoint = 'analytics/tag/' . rawurlencode( $meta['tag'] ) . '/detail';
 				break;
 			case 'community':
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This case's code block is the same as the block for the case on line 59. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1871)